### PR TITLE
Create the cnv bundle

### PR DESCRIPTION
**Pull request by naved001**
https://docs.openshift.com/container-platform/4.9/virt/install/installing-virt-cli.html#virt-subscribing-cli_installing-virt-cli

will put it in overlays at time of configuration since I dont want to deal with merge conflicts

### CODE REVIEW

File: cluster-scope/base/operators.coreos.com/subscriptions/openshift-cnv/subscription.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: hco-operatorhub
+  namespace: openshift-cnv
```
Comment (larsks) [inline cluster-scope/base/operators.coreos.com/subscriptions/openshift-cnv/subscription.yaml:5]: I think this should be named `kubevirt-hyperconverged` to match the name of the operator.

Comment (naved001) [reply cluster-scope/base/operators.coreos.com/subscriptions/openshift-cnv/subscription.yaml:5]: changed.

File: cluster-scope/base/operators.coreos.com/operatorgroups/openshift-cnv/operatorgroup.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: kubevirt-hyperconverged-group
```
Comment (larsks) [inline cluster-scope/base/operators.coreos.com/operatorgroups/openshift-cnv/operatorgroup.yaml:4]: The name of the parent directory should match the value of name: here in all cases.

Comment (naved001) [reply cluster-scope/base/operators.coreos.com/operatorgroups/openshift-cnv/operatorgroup.yaml:4]: okay, I'll update the directories. is this the name we are going with then? `kubevirt-hyperconverged-group` with the group suffix? Again, this is all from the redhat documentation and for whatever reason they chose these names.

Comment (naved001) [reply cluster-scope/base/operators.coreos.com/operatorgroups/openshift-cnv/operatorgroup.yaml:4]: well, if that's the name of the group then it has to be that.

**Review by larsks — APPROVED** [review]

**Pull request merged**